html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html" target="_blank">Home Page</a></li>
+      <li><a href="about-me.html">About Me</a></li>
+      <li><a href="../projects/projects.html" target="_blank">Projects</a></li>
+      <li>
+        <a href="../contact-us/contact-us.html" target="_blank">Contact Us</a>
+      </li>
```

Comment:
In this case, we don't want the `target="_blank"`, because all of these files are apart of our website. We would want to include this when we're sending users to a website that we don't control or own.

Reply:
okay copy that makes sense thank you